app/services/aspace/client.rb

```diff
@@ -29,5 +29,20 @@ def find_resource_uri(ead_id:)
         return uri if uri.present?
       end
     end
+
+    def find_top_container_uris(repository_uri:, ead_id:, indicators:)
+      query_params = []
+      query_params << ["q", "collection_identifier_u_stext:#{ead_id} indicator_u_icusort:[#{indicators.first} TO #{indicators.last}]"]
+
+      query_params << ["type[]", "top_container"]
+      query_params << ["fields[]", "uri"]
+      query_params << ["page", "1"]
```

Comment:
What happens when there are multiple pages of results, or is that not likely to happen?

Reply:
Right now it's not likely to happen.